docs/data/hubble/analyst-guide/end-to-end-analysis-example.mdx

```diff
@@ -0,0 +1,201 @@
+import add_column_chart from "/img/hubble/add-column-chart.png";
+import column_chart_config from "/img/hubble/column-chart-config.png";
+import resource_column_chart from "/img/hubble/resource-column-chart.png";
+
+This page will guide you through an end-to-end analysis use case with Hubble and [Google Looker Studio](https://cloud.google.com/looker-studio?hl=en)
+
+## Prerequisites
+
+1. Make sure you have connected to Hubble by following the instructions in the [Connecting](./connecting.mdx) page
+2. You have access to [Google Looker Studio](https://cloud.google.com/looker-studio?hl=en)
+3. You have read and understand the general [Best Practices](https://developers.stellar.org/docs/data/hubble/analyst-guide/optimizing-queries#best-practices) for querying BigQuery data
+
+## Create a Dashboard Analyzing Soroban Metrics
+
+### Attach Data Sources to Looker Studio
+
+1. Select `Create --> Data source`
+
+<img src={create_data_source} width="200" />
+
+2. Find and select the `BigQuery Google Connector`
+
+<img src={bq_connector} width="300" />
+
+3. Find the desired tables that you want to connect. For this example you will want to add a data source for:
+   1. `crypto-stellar.crypto_stellar.contract_data`
+
+<img src={add_contract_data} width="600" />
+
+### Create a New Report (Dashboard)
+
+1. Select `Create --> Report`
+
+<img src={create_report} width="200" />
+
+2. Add your data sources from above
+
+<img src={select_data_sources} width="300" />
+
+3. Insert a `Pie chart`
+
+<img src={add_pie_chart} width="200" />
+
+4. Choose `contract_data` as the `Data source`
+
+<img src={pie_contract_data} width="300" />
+
+5. Choose `closed_at` as the `Date Range Dimension` and `contract_durability` as the `Dimension`
+
+<img src={pie_config} width="300" />
+
+6. You should now have a pie chart showing the percentage of Temporary VS Persistent Contract Data Durability
+
+<img src={pie_chart_durability} width="500" />
+
+### Use Custom SQL to Create a Chart
+
+1. In your report, click `Add Data` which will be near the bottom right of your window
+
+<img src={click_add_data} width="500" />
+
+2. Select `BigQuery` and choose `CUSTOM QUERY` and select your desired `Billing Project` where the query will be charged
+
+<img src={add_custom_query} width="500" />
+
+3. Add the following query and click `Add`
+
+```sql
+-- Selects each ContractData ledger entry from the last 7 days from the contract_data table.
+with
+  contract_data as (
+    select
+      contract_id
+      , (contract_id || contract_key_type || contract_durability) as contract_entrykey
+      , date(closed_at) as ledger_date
+    from `crypto-stellar.crypto_stellar.contract_data`
+    where(
+      deleted is false
+      and (date(closed_at) between datetime_sub(current_datetime(), interval 7 day)
+      and current_datetime())
+    )
+  )
+
+-- Selects successful 'invoke_contract' Soroban transactions and its resource consumption from the enriched_history_operations table.
```

Comment:
This query is rather long and complex. We're joining `contract_data` (full history) on a `_current` table and also joining back to one of the enriched history operations example. While I like how we're moving into a more complex example, this feels like a leap and i think we would do better to have a slightly simpler example....maybe identifying expired entries on the network? or identifying which soroban operation types are most costly, etc. i can help brainstorm another example

Reply:
Yeah we can brainstorm and come up with a better example in our team meeting